src/streams/StreamItem.js

```diff
@@ -47,19 +47,30 @@ const componentStyles = createStyleSheet({
   muted: {
     opacity: 0.5,
   },
+  collapseIcon: {
+    marginRight: 8,
+  },
+  touchable: {
```

Comment:
nit: `pressable` would be a better name, given this is now a `Pressable` component and not a `Touchable`.